Dockerfile

```diff
@@ -9,25 +9,29 @@
 ARG USER=specter
 ARG DIR=/data/
 
-FROM python:3.10-slim-bullseye AS builder
+FROM python:3.10-bookworm AS builder
 
 ARG VERSION
 ARG REPO
 
-RUN apt update && apt install -y git build-essential libusb-1.0-0-dev libudev-dev libffi-dev libssl-dev rustc cargo libpq-dev
+RUN apt update && apt install -y git libusb-1.0-0-dev libudev-dev libffi-dev libssl-dev rustc cargo libpq-dev
```

Comment:
This is also just a build speed optimization, instead of using the slim image, just use the non-slim image during build, which has most packages we need for building preinstalled.